src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -132,6 +174,19 @@
   }
 }
 
+.pf-c-context-selector__toggle-button-icon {
+  position: relative;
+}
+
+.pf-c-context-selector__toggle-icon {
+  margin-right: var(--pf-c-context-selector__toggle-icon--MarginRight);
+  margin-left: var(--pf-c-context-selector__toggle-icon--MarginLeft);
+
+  .pf-c-context-selector.pf-m-top.pf-m-expanded & {
```

Comment:
We don't have a `.pf-m-top` class for context selector, so you don't need this block.

Reply:
If this isn't being used, can you remove it, too?